src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisWithActionsTest.java

```diff
@@ -869,75 +869,75 @@ private void testGeneratorAction(boolean dc, LoadFlowParameters.BalanceType bala
         parameters.setDistributedSlack(true).setBalanceType(balanceType);
         parameters.setDc(dc);
         OpenLoadFlowParameters openLoadFlowParameters = new OpenLoadFlowParameters();
-        openLoadFlowParameters.setSlackBusPMaxMismatch(slackBusPMaxMismatch);
+        openLoadFlowParameters.setSlackBusPMaxMismatch(1e-3);
         parameters.addExtension(OpenLoadFlowParameters.class, openLoadFlowParameters);
```

Comment:
There is really no reason to tolerate a larger difference between security analyses and manual modification in the network since they are supposed to be the same.
But you compare active power with a small resolution.
To ensure accurate results, use the Per Equation criteria type. This way, you avoid that errors accumulate on a terminal and cause your tests to tail, and know that for each node the error is bounded by MaxActivePowerMismatch
        openLoadFlowParameters.setNewtonRaphsonStoppingCriteriaType(NewtonRaphsonStoppingCriteriaType.PER_EQUATION_TYPE_CRITERIA);
        openLoadFlowParameters.setMaxActivePowerMismatch(1e-3);

Reply:
Is it ok now?

Reply:
yes ! Thanks for the modifcation.